superset/utils/log.py

```diff
@@ -47,10 +47,12 @@ def collect_request_payload() -> dict[str, Any]:
 
     payload: dict[str, Any] = {
         "path": request.path,
-        **request.form.to_dict(),
-        # url search params can overwrite POST body
-        **request.args.to_dict(),
     }
+    payload.update(**request.form.to_dict())
+    payload.update(**request.args.to_dict())
+    if request.is_json:
+        json_payload = request.get_json(cache=True, silent=True)
```

Comment:
NOTE: here I discovered 2 important things:
- the logging framework, that inspects the `flask.request` object to gather context, was missing out on `request.json` which in many cases has rich context to read from
- `request.json` is a stream and can only be read once reliably, after which it'll show as empty. Here using `get_json(cache=True)` ensures the process can read it later in the execution flow.